<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>1079</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="loadImgBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
background-color: #d35400;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;</string>
    </property>
    <property name="text">
     <string>Load image</string>
    </property>
   </widget>
   <widget class="QLabel" name="image1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>311</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #dddddd;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>311</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #dddddd;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>22</width>
      <height>431</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="image3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>100</y>
      <width>311</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #dddddd;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>550</y>
      <width>311</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #dddddd;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>550</y>
      <width>311</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #dddddd;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>550</y>
      <width>311</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #dddddd;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="threshBinaryCheckBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Thresh Binary</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="threshBinaryInvCheckBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Thresh Binary Inv</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="threshTruncCheckBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>526</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Thresh Trunc</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="treshTozeroRadioBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>526</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Thresh Tozero</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="treshTozeroInvRadioBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>526</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Thresh Tozero Inv</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>12</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>603</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
